src/main/kotlin/me/serce/solidity/ide/run/SearchUtils.kt

```diff
@@ -0,0 +1,20 @@
+package me.serce.solidity.ide.run
+
+import com.intellij.openapi.project.Project
+import com.intellij.psi.stubs.StubIndex
+import me.serce.solidity.lang.psi.SolContractDefinition
+import me.serce.solidity.lang.psi.SolFunctionDefinition
+import me.serce.solidity.lang.psi.SolNamedElement
+import me.serce.solidity.lang.stubs.SolGotoClassIndex
+
+
+object SearchUtils {
+  fun findContract(contractName: String?, project: Project): SolContractDefinition? {
```

Comment:
Given that `contractName` is supposed to be not-null (`!!` the code), `contractName ` should be non-nullable (`String`)

Reply:
Done